choreolib/src/main/java/choreo/auto/AutoChooser.java

```diff
@@ -45,7 +46,7 @@ public class AutoChooser implements Sendable {
   private final HashMap<String, Supplier<Command>> autoRoutines =
       new HashMap<>(Map.of(NONE_NAME, Commands::none));
 
-  private String selected = NONE_NAME;
+  protected String selected = NONE_NAME;
```

Comment:
Why was this changed to protected? If you want it published somewhere, then make AutoChoooser work with SmartDashboard directly, like SendableChooser does.

Also, SendableChooser provides a public getter `getSelected()`. Protected members are a code smell.